Makefile

```diff
@@ -11,12 +11,12 @@ export PATH := $(BIN):$(PATH)
 export GOBIN := $(abspath $(BIN))
 # Set to use a different Python interpreter. For example, `PYTHON=python make test`.
 PYTHON ?= python3
-CONFORMANCE_ARGS ?= --strict --strict_message --expected_failures=tests/conformance/nonconforming.yaml --timeout 10s
+CONFORMANCE_ARGS ?= --strict_message --expected_failures=tests/conformance/nonconforming.yaml --timeout 10s
```

Comment:
The `--strict` flag was removed in https://github.com/bufbuild/protovalidate/pull/334